src/util/DOMUtils.ts

```diff
@@ -96,13 +105,10 @@ export const getStringSize = (text: string | number, style: CSSProperties = {}):
     }
     // Need to use CSS Object Model (CSSOM) to be able to comply with Content Security Policy (CSP)
     // https://en.wikipedia.org/wiki/Content_Security_Policy
-    const measurementSpanStyle: Record<string, any> = { ...SPAN_STYLE, ...style };
-    Object.keys(measurementSpanStyle).map(styleKey => {
-      (measurementSpan.style as Record<string, any>)[styleKey] = measurementSpanStyle[styleKey];
-      return styleKey;
-    });
+    const measurementSpanStyle: Record<string, any> = { ...SPAN_STYLE, ...copyStyle };
+    Object.assign(measurementSpan.style, measurementSpanStyle);
 
-    measurementSpan.textContent = str;
+    measurementSpan.textContent = `${text}`;
```

Comment:
The template literal isn't used so can probably remove in favor of just text (though ik this was here before)

Reply:
need type casting if you don't convert it into a string.

Reply:
It is a string right? Would `String(text)` be more appropriate?

Reply:
That doesn't seem like a good idea in several ways. There's no need to box primitive types into objects, and it's performance-wise not optimal

Reply:
Hm, I don't like the fact that it's there 😅. But only because it wouldn't need to be there if the type was correct. Guess it's fine